### PR TITLE
Revise template descriptions

### PR DESCRIPTION
**Pull request by DelfinGala**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by vsnyc — APPROVED** [review]

**Comment by schottsfired** [comment]
Hi @DelfinGala , I noticed an issue in the screenshot in https://github.com/aws-quickstart/quickstart-snyk-security/pull/35#issuecomment-1177969131 (cc @nicarl) and it led me here.

Can we please revert the `SnykExternalId` parameter description updates in each template except the `ecr-auto` one? The `ecr-auto` template uses a group ID, whereas the three other templates use an organization ID. You are welcome to modify the lingo, but please use the parameter descriptions from https://github.com/aws-quickstart/quickstart-snyk-security/pull/32/commits/b226cb6febcbe99e246a1c0735c5fb1c05124565.

File: templates/snyk-ecr.template.yaml [hunk 1]
```diff
@@ -32,16 +32,16 @@ Metadata:
       SnykAWSAccountNumber:
         default: Snyk AWS Account ID
       ECRResourceARN:
-        default: ECR Resource ARN
+        default: Amaozon ECR resource ARN
 Parameters:
   SnykExternalId:
-    Description: Locate the organization ID by logging in to https://app.snyk.io and navigating to Settings. To integrate with multiple organizations, separate each ID with a comma.
+    Description: "Enter your Snyk group ID. To locate your group ID, log in to https://app.snyk.io, and navigate to account settings. For multiple organizations, separate each ID with a comma."
```
Comment (schottsfired) [inline templates/snyk-ecr.template.yaml:38]: here

File: templates/snyk-full.template.yaml [hunk 2]
```diff
@@ -48,33 +48,30 @@ Metadata:
         default: Quick Start S3 key prefix
 Parameters:
   SnykExternalId:
-    Description: Locate the organization ID by logging in to https://app.snyk.io and navigating to Settings. To integrate with multiple organizations, separate each ID with a comma.
+    Description: "Enter your Snyk group ID. To locate your group ID, log in to https://app.snyk.io, and navigate to account settings. For multiple organizations, separate each ID with a comma."
```
Comment (schottsfired) [inline templates/snyk-full.template.yaml:51]: here

File: templates/snyk-lambda.template.yaml [hunk 3]
```diff
@@ -27,18 +27,18 @@ Metadata:
...
       SnykAWSAccountNumber:
-        default: Snyk AWS Account ID
+        default: Snyk AWS account ID
       LambdaResourceARN:
-        default: Lambda Resource ARN
+        default: ARN for AWS Lambda resources
 Parameters:
   SnykExternalId:
-    Description: Locate the organization ID by logging in to https://app.snyk.io and navigating to Settings. To integrate with multiple organizations, separate each ID with a comma.
+    Description: "Enter your Snyk group ID. To locate your group ID, log in to https://app.snyk.io, and navigate to account settings. For multiple organizations, separate each ID with a comma."
```
Comment (schottsfired) [inline templates/snyk-lambda.template.yaml:35]: and here

**Pull request merged**